challenges-22-plus.yml

```
@@ -0,0 +1,192 @@
+lazy_frame_full_frame:
+  question: |
+    My animal rescue website has a page with very useful contact numbers that
+    people should use if they see a stray cat or dog. But, as usefull as this is,
+    I'd like to show this information at the bottom of every page!
+    
+    My footer looks like this:
+    
+    ```twig
+    <div id="footer">
+      <p>Emergency Contact Numbers:</p>
+
+      <turbo-frame id="contact-numbers" src="{{ path('contact_numbers') }}"></turbo-frame>
+    </div>
+    ```
+    
+    And my contact page has all of its content in the `body` block.
+    
+    ```twig
+    {% block body %}
+        <h1>Useful Contact Numbers</h1>
+        
+        <p>Here are some usefull contact numbers in case you are in an emergency and
```

```suggestion
        <p>Here are some useful contact numbers in case you are in an emergency and
```

We need to fix this in the answers too :)